upcase_booklist_app/upcase_booklist_app.py

```diff
@@ -54,5 +56,17 @@ def category(category_id):
     )
 
 
+@app.route("/authors/<int:author_id>")
+def author(author_id):
+    """Display a page about an author"""
+
+    author_object = None
+    for author in authors_data:
+        if author["author_id"] == author_id:
+            author_object = author
+
+    return render_template("author.html", author_object=author_object)
```

Comment:
Excellent job returning a single object that contains all the details relevant to an author!